src/mono/wasm/templates/templates/browser/.template.config/template.json

```diff
@@ -46,6 +46,7 @@
         }
       ],
       "defaultValue": "net8.0",
+      "replaces": "net7.0",
```

Comment:
Could you please explain this ?

Reply:
This finds and replaces instances of "net7.0" string in the project file, with `"net8.0"`.